essential-tests/src/testFixtures/kotlin/org/stellar/anchor/platform/e2etest/Sep24End2EndTests.kt

```diff
@@ -83,7 +83,6 @@ open class Sep24End2EndTests : AbstractIntegrationTests(TestConfig()) {
   ) = runBlocking {
     val keypair = SigningKeyPair.fromSecret(walletSecretKey)
     walletServerClient.clearCallbacks()
-    anchorReferenceServerClient.clearEvents()
```

Comment:
Is the endpoint used anymore? Should we remove it?

Reply:
Yeah, I was debating... There might be future use case. So, I kept it.

I will remove from the client and still keep the end-point.